func/ui.c

```diff
@@ -118,76 +111,25 @@ int requestRecipeNumber(){
     return recipeNumber -1;
 }
 
-/**
- * Prints information about a given recipe.
- */
-void printRecipe(const Recipe *recipe, const int people){
     /**
      * For mysterious reasons, the last digit of "cellLengths" becomes really big, and wrecks the system.
      * This issue is solved with a hack: add an extra item at the end of the array, and don't read it.
      */
-    int cellLengths[] = {22, 14, 14, 0};
-    const int numberOfCells = sizeof(cellLengths) / sizeof(cellLengths[0]) - 1;
+void printRecipe(Recipe *recipe, const int people){
```

Comment:
```suggestion
void printRecipe(const Recipe *recipe, const int people){
```
Hvorfor fjerner I en forbedring fra #38?

Reply:
koden virkede ikke for os men synes vi skal finde en løsning i fællesskab i morgen

Reply:
Ok, let's do that 👍